Sources/IO/IOAudioMixerByMultiTrack.swift

```diff
@@ -26,7 +26,7 @@ final class IOAudioMixerByMultiTrack: IOAudioMixerConvertible {
             }
             for id in tracks.keys {
                 buffers[id] = .init(outputFormat)
-                tracks[id] = .init(id: id, outputFormat: outputFormat)
+                tracks[id] = .init(id: id, outputFormat: outputFormat, delegate: self)
```

Comment:
[IMO]
- I remember encountering a bug in some version of Swift where specifying a delegate in the constructor in a Release build would cause the weak reference to become nil. I thought it would be better to set the delegate in this line.
- `tracks[id]?.delegate = self`

Reply:
Thanks, updated